Feature7 azure pipeline (#11)
data61/anonlink-client: README.md

# Anonlink Client

Client-face API to interact with anonlink system including command line tools and Rest API communication.
Anonlink system needs the following three components to work together:

* <a href='https://github.com/data61/clkhash'>clkhash</a>
* <a href='https://github.com/data61/blocklib'>blocklib</a>
* <a href='https://github.com/data61/anonlink-entity-service'>anonlink-entity-service</a>

This package provides an easy to use API to interact with the above packages to complete a record linkage job.

The way to interact with anonlink system is via Command Line Tool `anonlink`. You can hash data containing PII (Personal
 Identifying Information) locally using `anonlink hash`, generate candidate blocks locally to scale up record linkage 
 using `anonlink block`, create a record linkage job in entity service with `anonlink create-project` etc.

### Installation
Currently manual install:

```python3

> pip install git+https://https://github.com/data61/clkhash

Author: Should I release anonlink-client on Pypi manually for now to support pip install?

> pip install git+https://https://github.com/data61/clkhash

Reviewer: Sure

```

### Documentation
https://clkhash.readthedocs.io/en/stable/cli.html

### CLI Tool

> After installation, you should have a `anonlink` program in your path. Alternatively, you can use `client.cli`. For

Reviewer: We can't use the Python module name `client`. Suggest opening an issue.

Author: I made an issue #14 for this. Can you provide more details in the issue description? Do you mean `client` is not a recommended module name or you mean you cannot run `client.cli` in your machine?

Reviewer: Sure - done.

example, to hash PII data  `alice.csv` locally with schema `schema.json` and secret `horse`, run:
```bash
$ anonlink hash 'alice.csv' 'horse' 'schema.json' 'clk.json'
```

It will generate the CLK output and store in `clk.json`. To find out how to define the schema

> for your PII data, please refer <a href='https://clkhash.readthedocs.io/en/stable/schema.html'>this page</a> for 

Reviewer: ```suggestion
for your PII data, please refer [this page](https://clkhash.readthedocs.io/en/stable/schema.html) for 
```
